oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -0,0 +1,197 @@
+
+Responsibility of decentralized decision making
+-----------------------------------------------
+
+We assert that anyone can make an architectural decision, and that the only caveat is that they **must consult those who are meaningfully affected by the decision**, and those who have expertise in the area the decision is being taken.  The decision and all the advice they receive should be documented in an ADR, co-located with the relevant code following our established ADR guidelines in OEP-19.
+
+Encourage approachable decision making
+--------------------------------------
+
+The process of making a technical decision is not atomic, nor do many parts of it occur while writing a document about the decision.  Many decisions are *made* while writing code as we problem solve and navigate the idiosyncracies of the system.  The goal of our process should be to find a way of sharing and capturing those decisions where they impact others.  In this spirit, we encourage using "low-friction" artifacts - such as GitHub Issues - in those moments where drafting a full ADR document would be prohibitively distracting.  An issue is an easy place to gather quick, initial input, and naturally becomes resolved by a GitHub Pull Request - the ADR - which describes the decision in more detail.
+
+Decisions can have varying impacts and scopes
+---------------------------------------------
+
+The group of impacted individuals for a given decision may vary wildly, and some decisions will necessarily require more input than others.  A developer iterating on new "green field" code may not need to consult *anyone* except their immediate teammates, whereas a team making changes to a core data model may impact stakeholders and systems throughout the platform, meaning that there will necessarily be significant input into the decision.  The volume of input may make the latter decision harder to make.  That the complexity of the decision making process correlates with the impact of the decision indicates we're giving decisions due process.
+
+Decisions are an opportunity to learn and democratize knowledge
+---------------------------------------------------------------
+
+Engaging in public conversations around our decision making invites participation and creates learning opportunities.  An informed developer community who feels empowered to engage in the decision making process has numerous downstream benefits.  It deepens our well of expertise, increases retention for participating companies, provides opportunities for mentorship and upskilling, and ensures the process receives relevant and diverse input.
+
+This is an architectural decision making process
+------------------------------------------------
+
+This process is for architectural decisions.  It's not for product, user experience, pedagogical, legal, etc., decisions that are handled by other, non-engineering groups in our community.
+
+
```

Comment:
Recently OEP-1 was updated to define a Changelog, wherein you should mention your change (in this case, "Document Created") and link this PR. See https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#id3

Reply:
Ah, thank you!  I'll get that in.